defaults/parameters.yaml

```diff
@@ -11,6 +11,13 @@ strip_strain_prefixes:
   - SARS-CoV-2/
 
 sanitize_metadata:
+  metadata_id_columns:
+    - Virus name
+    - strain
+    - name
```

Comment:
Should probably keep the previous order of this default, since order is significant in the code.

Reply:
Good catch! I've fixed this in 79bc714